packages/babel-plugin-transform-flow-strip-types/src/index.js

```diff
@@ -57,7 +57,16 @@ export default function ({ types: t }) {
           node = node.expression;
         } while (t.isTypeCastExpression(node));
         path.replaceWith(node);
-      }
+      },
+
+      ImportSpecifier(path) {
```

Comment:
I think it'd be more consistent with what we do now, but yeah the current architecture is pretty tough to follow, no disagreement there. Definitely something we can fix.